design/design-doc-stacks.md

```diff
@@ -301,24 +414,25 @@ Possible solutions for package processing are listed below, along with their pro
   * Package images only contain the package contents, no processing logic is included in the package
   * The SM starts a job pod with an init container for the package image and copies all its contents to a shared volume in the job pod.  The Crossplane package processing logic runs in the main pod container and runs over the shared volume, sending all artifacts to `stdout`.
   * PRO: Package images are significantly smaller since they will only contain yaml files and icons, no binaries or libraries.
+  * PRO: The processing/unpacking logic can easily move to its own image that can be used as a base layer in the future (or a CLI tool), so this approach is not very divergent.
   * CON: The SM needs to understand package content and layout.
 * CLI tool
   * The unpacking logic could be built into a CLI tool that can be run over a package image.
   * PRO: This is the most versatile option as it can be used in contexts outside of a Crossplane cluster.
   * CON: Doesn't integrate very cleanly into the Crossplane mainline scenario with the SM.
 
-**Recommendation:** It is recommended to start with the "Job initContainer and shared volume" approach that copies the package contents from the package initContainer to a shared volume and has a Crossplane container main entry point perform the processing logic over the shared volume contents, sending out the artifacts to `stdout`.
-Stack packages should only have a few KB of yaml files, so it's a significant concern to blow that image size up to many MB right now.
-The processing/unpacking logic can easily move to its own image that can be used as a base layer in the future (or a CLI tool), so this approach is not very divergent.
-It is a good place to start and iterate over as we learn more without investing much effort that cannot be reused.
+Each of these designs offered a good place to start.  Through iteration over time we will learn more, hopefully without investing much effort that cannot be reused.
```

Comment:
I still think we shouldn't be removing the information about the path we chose to start with and that explains what the current code is doing.  We can update the wording in this PR to be more present tense (the implementation in the code base currently ...) and remove the language around "it is recommended".

But I think we shouldn't leave the reader hanging here by just saying they all offered a good place to start, but you're going to have to guess which one we chose! 😜

Reply:
In the second sentence of the parent section (Package Processing) I state:

> See the [Installation Flow](#installation-flow) for a more complete view of the current package processing implementation.

I think I covered (recovered?) the removed points throughout the rest of the document (especially earlier) but I added a few more touches in d5a809f4c5a98cf454e0a65339cf5107c3675bc2.

Does the flow still feel off?